docs/user/vsphere/install_upi.md

```diff
@@ -45,7 +45,7 @@ Memory
 Storage
 Networking
 
-[todo-link-to-minimum-resource-requirements]
+https://docs.openshift.com/container-platform/4.1/installing/installing_vsphere/installing-vsphere.html#minimum-resource-requirements_installing-vsphere
```

Comment:
the openshift docs use these docs as the source for content, so pointing back to them is weird.